lib/std/json/static.zig

```diff
@@ -307,32 +317,36 @@ fn internalParse(
             while (true) {
                 var name_token: ?Token = try source.nextAllocMax(allocator, .alloc_if_needed, options.max_value_len.?);
                 const field_name = switch (name_token.?) {
-                    .object_end => break, // No more fields.
                     inline .string, .allocated_string => |slice| slice,
-                    else => return error.UnexpectedToken,
+                    .object_end => { // No more fields.
+                        freeAllocated(allocator, name_token.?);
```

Comment:
`freeAllocated` does nothing for an `.object_end` token. We can remove this call.

Reply:
Fixed